c2corg_ui/templates/helpers/view.html

```diff
@@ -111,7 +111,8 @@
       title = ''
       if 'title_prefix' in locale and locale['title_prefix']:
           title = locale['title_prefix'] + ' : '
-      title += locale['title']
+      if 'title' in locale and locale['title']:
+          title += locale['title']
```

Comment:
I don't think this change is satisfaying because you stash the problem instead of solving it.

Isn't the problem here that the API does not return a ``title``? That does not look correct. Perhaps there's a bug in the API.
The API request for http://localhost:6555/routes/111829/fr/291699 (see the examples at https://github.com/c2corg/v6_ui/issues/1469#issuecomment-277222520) returns a ``locale`` with many lacking attribute:

![capture du 2017-02-08 13-12-12](https://cloud.githubusercontent.com/assets/1192331/22736806/682f7df0-ee00-11e6-96c5-bd144478693d.png)

The previous and next versions in http://localhost:6555/routes/history/111829/fr return all attributes as expected.

Why is the behaviour different for this version?
API issue?